README.md

```diff
@@ -43,12 +45,29 @@ algeria = Everypolitician.country('Algeria')
 national_assembly = algeria.legislature('Majlis')
 ```
 
-If you want to point at a different `countries.json`, e.g. a local path or a different url, you can set it like this:
+By default, the gem connects to EveryPolitician's data on GitHub over HTTPS and
+returns the most recent data. Specifically it uses the current index file,
+called `countries.json`, which itself contains links to specific versions of
+data files.
+
+If you want to point at a different `countries.json`, such as one you've
+already downloaded, you can override this default behavour by setting it like
+this:
 
 ```ruby
 Everypolitician.countries_json = 'path/to/local/countries.json'
 ```
 
+For more about `countries.json`, see [this description](http://localhost:4000/repo_structure.html).
+
+The data the gem provides is identical to that in the equivalent Popolo JSON
+file. You can download the JSON directly (from
```

Comment:
This part is slightly confusing. 

The data this gem provides is from the `countries.json` file, not from the Popolo JSON.

It can _also_ bridge you everypolitician-popolo, for dealing with the Popolo file, but if we're going to talk about that, then that needs to be much more explicit, especially as the usage we want to encourage there is to continue to call methods, not drop back into raw JSON.